client/coral-admin/src/graphql/fragments/commentView.graphql

```diff
@@ -12,4 +12,13 @@ fragment commentView on Comment {
       id
       title
     }
+    actions {
+      ... on FlagAction {
+        reason
+        message
```

Comment:
there is a separate `commentView.graphql` in `coral-framework/graphql/fragments`. The admin and the embed stream should not be sharing fragments if they are currently.